api/snippets-menu/py-cromwell-snippets-menu.json

```diff
@@ -3,7 +3,7 @@
     }
```

Comment:
the json in workbench did not have the latest changes (https://github.com/all-of-us/workbench-snippets/commit/9e0e7da5f72b01157891b0ed2ceda8a647fe9262) from workbench_snippets